### PR TITLE
May2023 corrections

### PR DESCRIPTION
**Pull request by rwaytes**
Added some extra functions (in read_layers, as well as amend_large_polygons and clean_vri_bem_output). Fixed the NA/"" issue.

@Karine-Pigeon, @nicgauthier Could someone review this?

### CODE REVIEW

**Comment by nicgauthier** [comment]
> Added some extra functions (in read_layers, as well as amend_large_polygons and clean_vri_bem_output). Fixed the NA/"" issue.
> 
> @Karine-Pigeon, @nicgauthier Could someone review this?

Yes!

**Comment by rwaytes** [comment]
> > Added some extra functions (in read_layers, as well as amend_large_polygons and clean_vri_bem_output). Fixed the NA/"" issue.
> > @Karine-Pigeon, @nicgauthier Could someone review this?
> 
> Yes!

Thank you!

File: R/amend_large_polygons.R [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+}
+}
+
+
+
+#extract large polygons
+extract_large_polygons <- function (vri_bem) {
+
+  #find polygons ovre 3500 ha
+  large_polygons <- vri_bem %>%
```
Comment (nicgauthier) [inline R/amend_large_polygons.R:57]: Just a suggestion but R now has a pipe operator included in base R  |> instead of %>%

**Review by nicgauthier — APPROVED** [review]
Very nice job Riley!

**Pull request merged**